forch/port_state_manager.py

```diff
@@ -188,10 +218,15 @@ def _handle_port_behavior(self, mac, port_behavior):
                 return
             state_machine.handle_port_behavior(port_behavior)
 
+    def _handle_unauthenticated_state(self, mac):
+        """Handle unauthenticated state"""
```

Comment:
no need for comment on private function (and it's highly redundant)

Reply:
removed